app/authenticated/account/template.hbs

```diff
@@ -0,0 +1,13 @@
+{{#page-layout/header as |header|}}
```

Comment:
Definitely! Looking at it now I should have added a `{{page-layout}}` "master" component that yields the header and content components, so the whole page layout is wrapped in a single outer component. Ah, hindsight... 😜